darwin/future/core/items/set_item_priority.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+from uuid import UUID
+
+from darwin.future.core.client import ClientCore
+from darwin.future.core.types.common import JSONType
+from darwin.future.data_objects.typing import UnknownType
+
+
+def set_item_priority(
+    api_client: ClientCore,
+    team_slug: str,
+    dataset_id: int,
+    item_id: UUID,
+    priority: int,
+    filters: Dict[str, UnknownType] = {},
+) -> JSONType:
+    """
+    Sets the priority of an item
+
+    Parameters
+    ----------
+    client: Client
+        The client to use for the request
+    team_slug: str
+        The slug of the team to set the priority for
+    priority: int
+        The priority to set
+
+    Returns
+    -------
+    JSONType
+    """
+    payload = {
+        "priority": priority,
+        "filters": {
+            "item_ids": [str(item_id)],
```

Comment:
I think the [set priority endpoint](https://docs.v7labs.com/reference/items-set-priority) can accept a list of `item_ids`, would it made sense to allow this to as well?

If so, maybe:
- `set_item_priority()` accepts `item_ids: List[UUID]`, then:
- `"item_ids": [str(item_id) for item_id in item_ids]` at this point